### PR TITLE
Create Pseudo-class-and-Pseudo-elements.md

### PR DESCRIPTION
**Pull request by AmrutaJayanti**
## Related Issue

Closes: #853 

## Description

I added `Pseudo-class-and Pseudo-elements.md`

## Type of PR

- [ ] Bug fix
- [ ] Feature enhancement
- [x] Documentation update
- [ ] Other (specify): _______________

## Checklist
- [x] I have performed a self-review of my code.
- [x] I have read and followed the Contribution Guidelines.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.

## Additional Context

[Include any additional information or context that might be helpful for reviewers.]

## Resources for Guidance

Here are some resources that may be helpful as you contribute to CodeHarborHub:
- [Code Harbor Hub Community Features](https://www.codeharborhub.live/community/features)
- [Docusaurus Documentation](https://docusaurus.io/docs/create-doc)
- [React.js Documentation](https://legacy.reactjs.org/docs/getting-started.html)
- [Markdown Guide](https://www.markdownguide.org/)
- [MDX Documentation](https://mdxjs.com/docs/)
- [Mermaid Documentation](https://mermaid.js.org/)

**Resources for DSA Features on Our Site:**

- Tabs: [Switch Languages](https://www.codeharborhub.live/community/features#multi-language-support-code-blocks)
- [Line Numbering](https://www.codeharborhub.live/community/features#line-numbering)
- [Math Equations](https://www.codeharborhub.live/community/features#math-equations-with-katex)

### CODE REVIEW

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
update your code based on suggestions

**Comment by AmrutaJayanti** [comment]
@Ajay-Dhangar I have made the changes . Please approve

File: docs/CSS/css-basics/Pseudo-class-and-Pseudo-elements.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+2. `:focus` : Applies when an element has received focus (e.g., when clicked on or tabbed to).
+
+```css
+input:focus {
+    border-color: blue;
+}
+```
+
+<BrowserWindow url="http://127.0.0.1:5500/index.html">
+<input type="text" required>
```
Comment (Ajay-Dhangar) [inline docs/CSS/css-basics/Pseudo-class-and-Pseudo-elements.md:39]: replace `<input type="text" required>` to `<input type="text" required />`

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
because this is mdx + React based so, you need self close you code because this is rule like xml for using react or mdx.

**Comment by AmrutaJayanti** [comment]
@Ajay-Dhangar I have made the requested changes. I apologize for any inconvenience caused. Thank you for your assistance in getting this PR merged.

**Review by Ajay-Dhangar — APPROVED** [review]
Done

**Pull request merged**